### PR TITLE
[Agent] Cleanup pkg-mgr code

### PR DESCRIPTION
**Pull request by again4you**
* Add the service-db code for future work.
* Change a c file into a cpp file to use the service-db instance.

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

File: daemon/pkg-mgr.cc [hunk 1]
```diff
@@ -40,6 +46,8 @@ static void _pkg_mgr_event_cb (const char *type, const char *package,
 
   if (event_type == PACKAGE_MANAGER_EVENT_TYPE_INSTALL && event_state == PACKAGE_MANAGER_EVENT_STATE_COMPLETED) {
     /* TODO Need to register the model into database */
+    db.connectDB ();
```
Comment (songgot) [inline daemon/pkg-mgr.cc:49]: I don't know if need a null check.
How about adding try{} catch{} ?

**Comment by jaeyun-jung** [comment]
LGTM, need rebase.

**Review by wooksong — APPROVED** [review]
LGTM

**Comment by again4you** [comment]
@anyj0527 already applied this patch to his work.

**Pull request closed without merging**